### PR TITLE
Updates, fixes and formatting docs related to the Kubernetes product

### PR DESCRIPTION
**Pull request by cortexx**
- Add missing warnings related to the billing of resources created in the Kubernetes Managed Service
- Improve formatting
- Add missing information related to the ports list to keep open when security groups are enabled on OpenStack side

### CODE REVIEW

File: pages/platform/kubernetes-k8s/getting-source-ip-behind-loadbalancer/guide.en-asia.md [hunk 1]
```diff
@@ -27,19 +27,24 @@ section: Tutorials
...
 
-**Last updated 14 June, 2021.**
+**Last updated 19<sup>th</sup> October 2021.**
 
 ## Before you begin
 
 This tutorial presupposes that you already have a working OVHcloud Managed Kubernetes cluster, and you have deployed there an application using the OVHcloud Managed Kubernetes LoadBalancer. If you want to know more on those topics, please look at the [using the OVHcloud Managed Kubernetes LoadBalancer](../using-lb/) documentation.
 
+> [!warning]
+> By deploying a __Network Load Balancer__ resource in your Managed Kubernetes Service, we will deploy a public Load Balancer allowing routing from the public traffic to your K8s cluster.
```
Comment (XavierDuthil) [inline pages/platform/kubernetes-k8s/getting-source-ip-behind-loadbalancer/guide.en-asia.md]:
```suggestion
> When a __LoadBalancer__ Service resource is created inside a Managed Kubernetes cluster, an associated Public Cloud Load Balancer is automatically created, allowing public access to your K8S application.
```

File: pages/platform/kubernetes-k8s/getting-source-ip-behind-loadbalancer/guide.en-asia.md [hunk 2]
```diff
@@ -27,19 +27,24 @@ section: Tutorials
...
+> This resource is hourly charged. For more information, please refer to the following documentation: [Network Load Balancer price](https://www.ovhcloud.com/en-ie/public-cloud/prices/#network)
```
Comment (XavierDuthil) [inline pages/platform/kubernetes-k8s/getting-source-ip-behind-loadbalancer/guide.en-asia.md]:
```suggestion
> The Public Cloud Load Balancer service is hourly charged and will appear in your Public Cloud project. For more information, please refer to the following documentation: [Network Load Balancer price](https://www.ovhcloud.com/en-ie/public-cloud/prices/#network)
```

File: pages/platform/kubernetes-k8s/persistent-volumes-on-ovh-managed-kubernetes/guide.en-au.md [hunk 3]
```diff
@@ -5,7 +5,17 @@ excerpt: 'Find out how to setup and manage Persistent Volumes on OVHcloud Manage
...
-**Last updated March 27<sup>th</sup>, 2020.**
+**Last updated 19<sup>th</sup>October 2021.**
+
+## Before you begin
+
+This tutorial presupposes that you already have a working OVHcloud Managed Kubernetes cluster, and some basic knowledge of how to operate it.  
+If you want to know more on those topics, please look at the [OVHcloud Managed Kubernetes Service Quickstart](../deploying-hello-world/).
+
+> [!warning]
+> By creating a __Persistent Volumes__ resource in your Managed Kubernetes Service, we will create for you a volume from the __Block Storage__ category.
```
Comment (XavierDuthil) [inline pages/platform/kubernetes-k8s/persistent-volumes-on-ovh-managed-kubernetes/guide.en-au.md]:
```suggestion
> When a __Persistent Volumes__ resource is created inside a Managed Kubernetes cluster, an associated Public Cloud __Block Storage__ volume is automatically created with it.
```

File: pages/platform/kubernetes-k8s/persistent-volumes-on-ovh-managed-kubernetes/guide.en-au.md [hunk 4]
```diff
@@ -5,7 +5,17 @@ excerpt: 'Find out how to setup and manage Persistent Volumes on OVHcloud Manage
...
+> This volume is visible through the OVHcloud Manager and is hourly charged. For more information, please refer to the following documentation: [Volume Block Storage price](https://www.ovhcloud.com/en-ie/public-cloud/prices/#storage)
```
Comment (XavierDuthil) [inline pages/platform/kubernetes-k8s/persistent-volumes-on-ovh-managed-kubernetes/guide.en-au.md]:
```suggestion
> This volume is hourly charged and will appear in your Public Cloud project. For more information, please refer to the following documentation: [Volume Block Storage price](https://www.ovhcloud.com/en-ie/public-cloud/prices/#storage)
```

**Review by XavierDuthil — CHANGES_REQUESTED** [review]
I also suggest adding the LB price warning to the https://docs.ovh.com/gb/en/kubernetes/deploying-an-application/ page, which makes the user create a LoadBalancer.

**Review by bs-ovh — APPROVED** [review]

**Review by XavierDuthil — APPROVED** [review]

**Comment by OVH-Team-Guides** [comment]
Thanks for your contribution @cortexx and @XavierDuthil. Deployment will be done soon.

**Pull request merged**